packages/database/src/migrations/20210927003701-LESMIS5YearOldEnrolmentRateMapOverlays-modifies-data.js

```diff
@@ -0,0 +1,201 @@
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const PERMISSION_GROUP = 'LESMIS Public';
+const OVERLAY_CODE = 'LESMIS_enrolment_ece';
+
+const OVERLAY_GROUPS = [
+  {
+    parentCode: 'Root',
+    children: [
+      {
+        name: 'ECE/Pre-Primary Enrolment',
+        code: `${OVERLAY_CODE}_Group`,
+      },
+    ],
+  },
+  {
+    parentCode: `${OVERLAY_CODE}_Group`,
+    children: [
+      {
+        name: 'District Level',
+        code: `${OVERLAY_CODE}_District_Group`,
+      },
+    ],
+  },
+];
+
+const MAP_OVERLAYS = [
+  {
+    parentCode: `${OVERLAY_CODE}_District_Group`,
+    children: [
+      {
+        reportCode: `${OVERLAY_CODE}_5_priority_District_map`,
+        name: 'Enrolment 5 year olds (40 priority districts)',
+        dataElements: ['er_district_ece_5_t'],
+        sortOrder: 3,
+      },
+    ],
+  },
+];
+
+const getReport = (reportCode, dataElements) => ({
+  code: reportCode,
+  config: {
+    fetch: {
+      dataElements,
+      aggregations: [
+        {
+          type: 'FINAL_EACH_YEAR',
+          config: {
+            aggregationEntityType: 'requested',
+            dataSourceEntityType: 'sub_district',
+            dataSourceEntityFilter: {
+              attributes_type: 'LESMIS_Target_District',
+            },
+          },
+        },
+      ],
+    },
+    transform: [
+      {
+        transform: 'updateColumns',
+        insert: {
+          organisationUnitCode: '=$organisationUnit',
+          value: '=divide($value, 100)',
+        },
+        exclude: ['organisationUnit', 'dataElement', 'period'],
+      },
+    ],
+  },
+});
+
+const getMapOverlay = (name, reportCode) => ({
+  id: reportCode,
+  name,
+  userGroup: PERMISSION_GROUP,
+  dataElementCode: 'value',
+  isDataRegional: true,
+  legacy: false,
+  report_code: reportCode,
+  measureBuilder: 'useReportServer',
+  measureBuilderConfig: {
+    dataSourceType: 'custom',
+    reportCode,
+  },
+  presentationOptions: {
+    scaleType: 'performance',
+    displayType: 'shaded-spectrum',
+    measureLevel: 'SubDistrict',
+    valueType: 'percentage',
+    scaleBounds: {
+      left: {
+        max: 'auto',
+      },
+      right: {
+        min: 'auto',
+      },
+    },
+    periodGranularity: 'one_year_at_a_time',
+  },
+  countryCodes: '{"LA"}',
+  projectCodes: '{laos_schools}',
+
+  legacy: false,
+  report_code: reportCode,
+});
+
+const addMapOverlayGroup = async (db, parentCode, { name, code }) => {
+  const parentId = await codeToId(db, 'map_overlay_group', parentCode);
+
+  const overlayGroupId = generateId();
+  await insertObject(db, 'map_overlay_group', { id: overlayGroupId, name, code });
+
+  return insertObject(db, 'map_overlay_group_relation', {
+    id: generateId(),
+    map_overlay_group_id: parentId,
+    child_id: overlayGroupId,
+    child_type: 'mapOverlayGroup',
+  });
+};
```

Comment:
Maybe get rid of these helper functions alongside deleting removing the `OVERLAY_GROUPS`